www-client/cromite-bin/cromite-bin-139.0.7258.128.ebuild

```diff
@@ -0,0 +1,167 @@
+# Copyright 2009-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+CHROMIUM_LANGS="af am ar as az be bg bn bs ca cs cy da de el en-GB es es-419 et eu fa fi fil
+	fr fr-CA gl gu he hi hr hu hy id is it ja ka kk km kn ko ky lo lt lv mk ml mn mr ms my
+	nb ne nl or pa pl pt-BR pt-PT ro ru si sk sl sq sr sr-Latn sv sw ta te th tr uk ur uz
+	vi zh-CN zh-HK zh-TW zu"
+
+inherit chromium-2 desktop readme.gentoo-r1 xdg-utils
+
+DESCRIPTION="Cromite a Bromite fork with ad blocking and privacy enhancements; take back your browser!"
+HOMEPAGE="https://github.com/uazo/cromite"
+CROMITE_COMMIT_ID="4b8cc71d7b35b86cf1789961d727dc4c19d9b28d"
+SRC_URI="https://github.com/uazo/cromite/releases/download/v${PV}-${CROMITE_COMMIT_ID}/chrome-lin64.tar.gz"
```

Comment:
Please either use `CROMITE_COMMIT_ID` or `PF` in file name, like this: ` -> ${PN}-${CROMITE_COMMIT_ID}.tar.gz`